cmd/device-worker/main.go

```diff
@@ -100,32 +100,32 @@ func main() {
 
 	// --- Client metrics configuration ---
 	metricsStore, err := metrics.NewTSDB(dataDir)
-	configManager.RegisterObserver(metricsStore)
+	configManager.RegisterObserver(metricsStore, true)
```

Comment:
I will fix `TSDB` later not to be intialized with defaults in the factory method, but on the first update.